scheds/rust/scx_rusty/src/main.rs

```diff
@@ -144,6 +147,12 @@ struct Opts {
     #[clap(short = 'K', long, default_value = "100.0")]
     kick_greedy_under: f64,
 
+    /// When the system's utilization is lower than this, ignore weights and
+    /// use only duty to make load balancing decisions. Advised to be at least
+    /// 100.0 to accommodate proper load distribution.
+    #[clap(short = 'L', long, default_value = "110.0")]
+    lb_dcycle_under: f64,
```

Comment:
Does this need to be a user-visible parameter? When duty cycle sum is lower than what's needed to derive the solution, the algorithm actually is giving the right answer that everyone needs to be adjusted to the same weight. So, we can skip it if we know that there's gonna be no answer but I don't see what control users would want to have over it.

Reply:
I agree. I added this based on prior conversations we had where I was under the impression that we wanted this to be similar to greedy threshold, etc. I don't think it makes sense for infeasible weights so I'll remove.